mlflow/azureml/__init__.py

```diff
@@ -312,18 +503,65 @@ def _get_mlflow_azure_resource_name():
     return resource_prefix + unique_id
 
 
+def _get_mlflow_azure_name_from_run(run_id):
+    """
+    :return: A unique name for an Azure resource indicating that the resource was created by
+             MLflow
+    """
+    azureml_max_resource_length = 32
+    resource_prefix = "mlflow-run-"
+    suffix = run_id[:azureml_max_resource_length-len(resource_prefix)]
+    return resource_prefix + suffix
+
+
+def _create_mlflow_wheel(mlflow_dir, out_dir):
+    """
+    Create the wheel of MLFlow by using setup.py bdist_wheel in the outdir.
+
+    :param model_path: The absolute path to the mlflow.
+    :param out_dir: The absolute path to the outdir.
+    :return: The absolute path to the wheel.
+    """
+    unresolved = Path(out_dir)
+    unresolved.mkdir(parents=True, exist_ok=True)
+    out_path = unresolved.resolve()
+    subprocess.run([sys.executable, "setup.py", "bdist_wheel", "-d", out_path],
+                   cwd=mlflow_dir, check=True)
+    files = list(out_path.glob("./*.whl"))
+    if len(files) < 1:
+        raise MlflowException("Error creating MLFlow Wheel - couldn't"
+                              " find it in dir {} - found {}".format(out_path, files))
+    if len(files) > 1:
+        raise MlflowException(
+            "Error creating MLFlow Wheel - couldn't"
+            " find it in dir {} - found several wheels {}".format(out_path, files))
+    return files[0]
+
+
+def parse_deploy_config(json_dict_input):
+    from azureml.core.webservice import AciWebservice, AksWebservice
+    computes = {"aks": AksWebservice, "aci": AciWebservice}
+    try:
+        compute_target_type = json_dict_input["computeType"].lower()
+        del json_dict_input["computeType"]
+        return computes[compute_target_type].deploy_configuration(**compute_target_type)
```

Comment:
Sorry, I don't understand what the `**compute_target_type` doing here?

Reply:
ah, that's a mistake, should be **json_dict_input